activerecord/lib/active_record/connection_adapters/postgresql/quoting.rb

```diff
@@ -193,7 +193,7 @@ def lookup_cast_type_from_column(column) # :nodoc:
 
         private
           def lookup_cast_type(sql_type)
-            super(query_value("SELECT #{quote(sql_type)}::regtype::oid", "SCHEMA").to_i)
+            super(query_value("SELECT #{quote(sql_type)}::regtype::oid", "SCHEMA", allow_retry).to_i)
```

Comment:
Typo?

Reply:
Yes thank you, this was super tedious 😓 I double checked all of the others and I think this is the only one